react/src/components/Table/Table.tsx

```diff
@@ -224,13 +239,19 @@ const Table: React.FC<TableProps> = ({ variantData }) => {
                         accessor: 'individualId',
                         id: 'individualId',
                         Header: 'Individual ID',
-                        width: getColumnWidth(tableData, 'individualId', 'Individual ID'),
+                        width: getColumnWidth(
+                            Math.max(...tableData.map(row => (row.individualId || '').length)),
```

Comment:
If we are calling `Math.max` for every time we call `getColumnWidth`, can this also be part of the `getColumnWidth` function?

Reply:
Yes for sure, this can also be a part of `getColumnWidth` function. I didn't include it in `getColumnWidth` function because we would pass a large size parameter to `getColumnWidth`.